pkg/annotations/ingress_annotations.go

```diff
@@ -141,6 +146,17 @@ func BackendProtocol(ing *v1beta1.Ingress) (ProtocolEnum, error) {
 	return HTTP, NewInvalidAnnotationContent(BackendProtocolKey, protocol)
 }
 
+// Get extended hostNames for a given hostname
+func GetHostNameExtensions(ing *v1beta1.Ingress, hostname string) ([]string, error) {
```

Comment:
`hostname` field is not used in the function.

Reply:
Will remove. This was remnant of older design